supports/integration-test/reset_env.py

```diff
@@ -73,6 +73,27 @@ def test_create_10M_DFSIO(self):
             subprocess.call("hdfs dfs -mv /benchmarks/TestDFSIO/io_data " +
                             TEST_DIR + str(i) + "_data")
 
+
+    def test_create_10K_0KB_DFSIO_parallel(self):
```

Comment:
I think each client should create several dirs, each dir has 10,000~100,000 files. So, maybe you need another loop.